app/http/controllers/ExerciseController.ts

```diff
@@ -2,23 +2,34 @@ import { NextFunction, Request, Response } from "express";
 import { ExerciseService } from "../../services/ExerciseService";
 
 export const ExerciseController = {
-    /**
-      * Handles the fetchData request
-      * 
-      * @param req {Request} Express request object
-      * @param res {Response} Express response object
-      * @param next {NextFunction} Express NextFunction (used for middleware)
-      * @throws {NotFoundException} File is Not found
-      */
-    getInfo(req: Request, res: Response, next: NextFunction) {
-        try {
-            const dat = ExerciseService.fetchData(req.params.id)
-            res.send(dat);
-        } catch (e) {
-            return next(e);
-        }
+  /**
+   * Handles the fetchData request
+   *
+   * @param req {Request} Express request object
+   * @param res {Response} Express response object
+   * @param next {NextFunction} Express NextFunction (used for middleware)
+   * @throws {NotFoundException} File is Not found
+   */
+  getInfo(req: Request, res: Response, next: NextFunction) {
+    try {
+      const dat = ExerciseService.fetchData(
+        req.params.exerciseID,
+        req.params.id
+      );
+      res.send(dat);
+    } catch (e) {
+      return next(e);
+    }
+  },
 
-    },
-
-
-}
\ No newline at end of file
+  getConst(req: Request, res: Response, next: NextFunction) {
+    try {
+      const dat = ExerciseService.getStatus(
```

Comment:
Change the constant name of dat. It is vague on what data it is actually holding